.circleci/config.yml

```diff
@@ -41,8 +42,15 @@ jobs:
 
       - run: *install_jdk
 
+      - run:
+          name: Debug
+          command: |
+            echo $MAVEN_OPTS | base64
+
       - run:
           name: Build presto
+          # environment:
+          #   MAVEN_OPTS: "true"
```

Comment:
Debug step above and these comments can be dropped?

Reply:
Of course. Sorry, I forgot to delete.

Reply:
@takezoe Fixed. PTAL.